crates/modelardb_server/src/metadata/mod.rs

```diff
@@ -1105,6 +1406,494 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn test_save_compressed_file_for_column_in_existing_model_table() {
+        let compressed_file = CompressedFile::new(Uuid::new_v4(), 100, 200, 37.0, 73.0);
+
+        // An assert is purposely not used so the test fails with information about why it failed.
+        create_metadata_manager_with_named_model_table_and_save_files(
+            "model_table",
+            &[compressed_file],
+        )
+        .await
+        .unwrap();
+    }
+
+    #[tokio::test]
+    async fn test_save_compressed_file_for_column_in_missing_model_table() {
+        let compressed_file = CompressedFile::new(Uuid::new_v4(), 100, 200, 37.0, 73.0);
+        assert!(
+            create_metadata_manager_with_named_model_table_and_save_files(
+                "table_model",
+                &[compressed_file]
+            )
+            .await
+            .is_err()
+        );
+    }
+
+    #[tokio::test]
+    async fn test_save_compressed_file_with_swaped_timestamps_for_column_in_existing_model_table() {
+        let compressed_file = CompressedFile::new(Uuid::new_v4(), 200, 100, 37.0, 73.0);
+        assert!(
+            create_metadata_manager_with_named_model_table_and_save_files(
+                "model_table",
+                &[compressed_file]
+            )
+            .await
+            .is_err()
+        );
+    }
+
+    #[tokio::test]
+    async fn test_save_compressed_file_with_swaped_values_for_column_in_existing_model_table() {
```

Comment:
Should it be `swapped` instead? Maybe just replace it with `invalid`.

Reply:
Fixed by changing it to invalid.